server/test/com/cloud/network/firewall/FirewallManagerTest.java

```diff
@@ -71,6 +79,7 @@
 //        super.setUp();
 //    }
 
+    @Ignore("Requires database to be set up")
```

Comment:
@wilderrodrigues @borisroman I cannot judge this. If it has no value, we can leave it out. Your call.